introduction.md

```diff
@@ -0,0 +1 @@
+Greetings, I am called Richard Carls. Please teach me in the ways of git-fu!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.